storybook/stories/api/offset.js

```diff
@@ -1,39 +0,0 @@
-// @flow
```

Comment:
Do you think there is value in keeping this example but updating to use `<Offset />`?